modules/caldav-sync/caldav-sync-web/bnd.bnd

```diff
@@ -1,13 +1,10 @@
 Bundle-Activator: it.smc.calendar.caldav.sync.listener.Activator
-
 Bundle-Copyright: SMC Treviso
 Bundle-DocURL: http://www.smc.it
 Bundle-License: LGPL
-
 Bundle-Name: CalDAV Sync Web
 Bundle-SymbolicName: it.smc.calendar.caldav
-Bundle-Version: 2.1.4
-
+Bundle-Version: 2.2.0
```

Comment:
Why a minor version bump? I don't see critical changes on the exported classes

Reply:
Ok, I'm going to fix that